ui/public/config.json

```diff
@@ -1,5 +1,12 @@
 {
   "apiBase": "/client/api",
+  "servers": [
```

Comment:
I think this feature shouldn't be enabled by default, wouldn't it confuse users/admins? @utchoang @davidjumani @wido @nvazquez @sureshanaparti @weizhouapache @Pearl1594 @shwstppr ?

Reply:
If the list only shows one server at the time, what would it confuse them? There is only one option to select

Reply:
@rhtyd Yes. I will set it to default hidden mode. If the user/admin wants to enable it, the configuration can be changed.

Reply:
Looks good with @utchoang's change, disabled by default. +1 for getting this in asap

Reply:
+1
thanks @utchoang